README.md

```diff
@@ -6,18 +6,18 @@ Redis::ScriptCache - Cached Lua scripts on a Redis server
 
     use Redis;
     use Redis::ScriptCache;
-    
+
```

Comment:
Blank lines shouldn't result in separate code blocks in the README files. Unless I accidentally removed it, there was nothing set up in dist.ini to build the README. So, I updated dist.ini to build both README.md for Github and a standard README for the distribution. I also dropped MANIFIEST.SKIP. It shouldn't be necessary with Git::GatherDir. See 7e625b4.